hw3/tests/test_hw3_t03.py

```diff
@@ -0,0 +1,121 @@
+from hw3.hw3_t03 import make_filter
+
+
+def test_polly():
```

Comment:
Who is Polly?   

This is no a filter for characters, it is a general-purpose filter. Characters are used as an example.

You have key, value, and item, so please build your positive and negative test around these terms.